app/v2/notifications/notification_schemas.py

```diff
@@ -122,7 +122,29 @@
     "definitions": {
         "notification": get_notification_response
     },
+}
 
+send_a_file_validation = {
+    "type": "object",
+    "additionalProperties": False,
+    "properties": {
+        "file": {
+            "type": "string",
+            "required": True,
+        },
+        "is_csv": {
+            "type": ["boolean", "null"],
+            "required": False,
+        },
+        "confirm_email_before_download": {
+            "format": "send_a_file_confirm_email",
```

Comment:
```suggestion
            "format": "send_a_file_confirm_email",
            "type": ["boolean", "null"],
```

Do we want to use jsonschema's type checking here? We already do with `is_csv`. (Depending on what we decide to do with the error messages at least)

Reply:
No because then you lose control over the error message. But this comment is probably overlapping upon the conversation above so let's keep it there 👍